doc/en/user/source/community/opensearch-eo/configuration.rst

```diff
@@ -130,6 +130,14 @@ geometry (mind, the output must be forced not to be escaped, using ``?no_esc``,
 as ``minx``, ``miny``, ``maxx``, ``maxy`` that can be used to extract the bounding box
 corner values. All these function expect a geometry as input.
 
+
+The templates can use a ``parseJSON`` function that takes json file location and parse that json file in the free marker templates. The path should be relative to the geoserver data directory. The function returns a string JSON that can be parsed using the ?eval_json function.
+
+:code:`<#assign loadedJSON = "${parseJSON('readAndEval.json')}">`
+:code:`<#assign evalJSON = loadedJSON?eval_json>`
```

Comment:
```suggestion
Finally templates can use a ``loadJSON`` function to read a JSON from a file inside the GeoServer data directory. 
The path to the JSON file can be absolute eg. :code:`"${loadJSON('/path/to/read.json')}"`, or a plain file name, in case the JSON file is present in the GeoServer root directory eg. :code:`"${loadJSON('read.json')}"`.
The function returns a string JSON that can be parsed using the  :code:`?eval_json` free marker built-in function:

:code:`<#assign loadedJSON = "${parseJSON('readAndEval.json')}">`
:code:`<#assign evalJSON = loadedJSON?eval_json>`
```

Reply:
done